parlai/core/build_data.py

```diff
@@ -495,7 +495,6 @@ def modelzoo_path(datapath, path):
                 animal_ = '.'.join(animal.split(".")[:-1]) + '.build'
                 module_name_ = 'parlai.zoo.{}'.format(animal_)
                 my_module = importlib.import_module(module_name_)
-                my_module.download(datapath)
```

Comment:
what's the motivation for this change?

Reply:
+1, this feels wrong

Reply:
Was a local change that I forgot to remove. Thank you for catching it! Reverted this part